composer.json

```diff
@@ -28,7 +28,7 @@
     ],
     "require": {
         "php": "8.1.*||8.2.*||8.3.*||8.4.*",
-        "composer/composer": "^2.8.5",
+        "composer/composer": "dev-main",
```

Comment:
We should probably pin to the next stable Composer release, when that happens :+1:

Reply:
^2.8.6@dev

Reply:
Updated to use `^2.8.6` now it is released :)